src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -169,16 +175,16 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
           Other users will be able to reply to your prompt and direct message you.
         </Text>
         <TouchableOpacity onPress={toggleSwitchAnon}>
-          <CheckBoxCustom value={isAnonymity} label="Allow anonymous messages?" />
+          <CheckBoxCustom value={isAnonymity} label="Allow messages from incognito users?" />
         </TouchableOpacity>
 
         {isAnonymity && (
           <>
-            <Divider style={{marginVertical: 12, backgroundColor: COLORS.gray}} />
+            <Divider style={{marginVertical: 6, backgroundColor: COLORS.gray}} />
             <TouchableOpacity onPress={toggleSwitchAnonAllowFollowing}>
               <CheckBoxCustom
                 value={isAllowFollowingSendDM}
-                label="Only allow anon DMs from users you follow?"
+                label="Allow incognito only from users you follow?"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in checkbox labels from "anonymous" to "incognito" is consistent and clear. This change improves the clarity of the setting's purpose for users.
```diff
- <CheckBoxCustom value={isAnonymity} label="Allow anonymous messages?" />
+ <CheckBoxCustom value={isAnonymity} label="Allow messages from incognito users?" />

- label="Only allow anon DMs from users you follow?"
+ label="Allow incognito only from users you follow?"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->